WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/NewSiteCreationVerticalsViewModel.kt

```diff
@@ -172,15 +180,16 @@ class NewSiteCreationVerticalsViewModel @Inject constructor(
                 }
             }
         } else {
-            showFullscreenConnectionErrorWithDelay(query)
+            showConnectionErrorWithDelay(query)
         }
     }
 
-    private fun showFullscreenConnectionErrorWithDelay(query: String) {
+    private fun showConnectionErrorWithDelay(query: String) {
         updateUiStateToContent(query, Loading(Ready(emptyList()), false))
         launch {
             // We show the loading indicator for a bit so the user has some feedback when they press retry
             delay(CONNECTION_ERROR_DELAY_TO_SHOW_LOADING_STATE)
+            tracker.trackErrorShown(ERROR_CONTEXT, INTERNET_UNAVAILABLE_ERROR, "List item error")
```

Comment:
Can we add a constant for `List item error`, preferably in `NewSiteCreationTracker` as explained in an earlier comment?